modules/packages/apt_get

```diff
@@ -166,7 +166,7 @@ def list_updates(online):
         if result != 0:
             return result
 
-    process = subprocess_Popen([apt_get_cmd] + apt_get_options + ["-s", "upgrade"], stdout=subprocess.PIPE)
+    process = subprocess_Popen([apt_get_cmd] + apt_get_options + ["--simulate", "--ignore-hold", "upgrade"], stdout=subprocess.PIPE)
```

Comment:
:+1: for using the long option! I think the `--ignore-hold` option deserves a comment though.